packages/react-instantsearch/src/core/highlight.test.js

```diff
@@ -1,101 +1,112 @@
 /* eslint-env jest, jasmine */
-import React from 'react';
+import {parseAlgoliaHit, parseHighlightedAttribute} from './highlight.js';
```

Comment:
> export default function highlight

There is a benefit in having both function namely exported: it explicitly state what are each functions. Also there is not real hierarchy between them. So I would stay like that, especially because this is not an API to be used outside of the library.

> only test highlight()

Having two functions allows for finer grained tests and makes responsibility of each functions  more explicit. If tests on `parseHighlightedAttribute` do not fail but tests on `parseAlgoliaHit` fail, we know that the error is on `parseAlgoliaHit` for sure. Otherwise it means that we have to analyze both functions to know the source of the error.